common/static/coffee/src/discussion/views/discussion_thread_list_view.coffee

```diff
@@ -243,13 +243,14 @@ if Backbone?
       @trigger("thread:removed", thread_id)
 
     setActiveThread: (thread_id) ->
+      @$(".forum-nav-thread-link").find(".sr").remove()
       @$(".forum-nav-thread[data-id!='#{thread_id}'] .forum-nav-thread-link").removeClass("is-active")
-      @$(".forum-nav-thread[data-id='#{thread_id}'] .forum-nav-thread-link").addClass("is-active")
+      @$(".forum-nav-thread[data-id='#{thread_id}'] .forum-nav-thread-link").addClass("is-active").find(".forum-nav-thread-wrapper-1").prepend('<span class="sr">' + gettext("Current conversation") + '</span>')
 
     goHome: ->
       @template = _.template($("#discussion-home").html())
       $(".forum-content").html(@template)
-      $(".forum-nav-thread-list a").removeClass("is-active")
+      $(".forum-nav-thread-list a").removeClass("is-active").find(".sr").remove()
```

Comment:
The `.is-active` class here was pre-existing and is set when a conversation/topic is chosen. I'm then looking for the active topic and adding `.sr` only span with specific content (i18n'd, of course). This line looks for the `.sr` and removes it when making a topic _not_ active.

Let's discuss, because maybe I'm not following?